doc/release/RELEASE-NOTES.md

```diff
@@ -34,6 +34,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Bugfix
 * [OSDEV-1048](https://opensupplyhub.atlassian.net/browse/OSDEV-1048) - Fixed error "User Cannot read properties of undefined (reading 'length')".
 * [OSDEV-1180](https://opensupplyhub.atlassian.net/browse/OSDEV-1180) - Introduced a 10,000-download limit check on the api/facilities-downloads API endpoint to prevent non-API users from downloading more than 10,000 production locations.
+* [OSDEV-1178](https://opensupplyhub.atlassian.net/browse/OSDEV-1178) - Make follow-up fix for the "Something went wrong" error after clicking on Dashboard -> View Facility Claims.
```

Comment:
Nit:
It sounds a little abstract. I think it would be helpful to add a short statement about the root cause of the problem.

Reply:
Done, pls re-review